webapp/store/views.py

```diff
@@ -261,7 +247,16 @@ def publisher_details(publisher):
         publisher_content_path = flask.current_app.config["CONTENT_DIRECTORY"][
             "PUBLISHER_PAGES"
         ]
-        context = _get_file(publisher_content_path + publisher + ".yaml")
+
+        context = helpers._get_file(
+            publisher_content_path + publisher + ".yaml"
+        )
+
+        "/{}{}.yaml".format(
+            flask.current_app.config["CONTENT_DIRECTORY"]["PUBLISHER_PAGES"],
+            publisher,
+            ".yaml",
```

Comment:
You can remove the `.yaml` argument.

Reply:
This is still relevant, no? This file hasn't been changed?

Reply:
I think we can actually remove 256-259, does not seem to do anything.